docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-web-5.2020.6.adoc

```diff
@@ -0,0 +1,344 @@
+[[tck-results]]
+= TCK Results
+
+As required by the
+https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License],
+following is a summary of the TCK results for releases of Jakarta EE Platform, Web Profile.
+
+**Payara Server 5.2020.6 Web Profile Certification Summary**
+
+- Product Name, Version and download URL (if applicable):
++
+https://payara.fish/download[Payara Server 5.2020.6]
```

Comment:
We need to use the compatibility logo to be correct legally (so other TCK result pages). Also, download link should point community page https://www.payara.fish/downloads/payara-platform-community-edition/

Reply:
Surely you can't use the logo until certified?

Reply:
I think that the product name should also be `Payara Server 5.2020.6 (Web Profile)`, which is used on the download page.